certified-connectors/bitskout/apiDefinition.swagger.json

```diff
@@ -163,7 +163,737 @@
           }
         ]
       }
-    }
+    },
+    "/actions/invoices": {
+			"post": {
+				"responses": {
+					"200": {
+						"description": "default",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"outputs": {
+									"type": "object",
+									"properties": {
+										"CURRENCY": {
+											"type": "string",
+											"description": "CURRENCY"
+										},
+										"CUSTOMER_ADDRESS": {
+											"type": "string",
+											"description": "CUSTOMER_ADDRESS"
+										},
+										"CUSTOMER_NAME": {
+											"type": "string",
+											"description": "CUSTOMER_NAME"
+										},
+										"DISCOUNT": {
+											"type": "string",
+											"description": "DISCOUNT"
+										},
+										"DUE_DATE": {
+											"type": "string",
+											"description": "DUE_DATE"
+										},
+										"INVOICE_RECEIPT_DATE": {
+											"type": "string",
+											"description": "INVOICE_RECEIPT_DATE"
+										},
+										"INVOICE_RECEIPT_ID": {
+											"type": "string",
+											"description": "INVOICE_RECEIPT_ID"
+										},
+										"LINE_ITEMS": {
+											"type": "string",
+											"description": "LINE_ITEMS"
+										},
+										"NUMBER_OF_PAGES": {
+											"type": "integer",
+											"format": "int32",
+											"description": "NUMBER_OF_PAGES"
+										},
+										"RECEIVER_ADDRESS": {
+											"type": "string",
+											"description": "RECEIVER_ADDRESS"
+										},
+										"RawJSON": {
+											"type": "string",
+											"description": "RawJSON"
+										},
+										"SUBTOTAL": {
+											"type": "string",
+											"description": "SUBTOTAL"
+										},
+										"SUPPLIER_ADDRESS": {
+											"type": "string",
+											"description": "SUPPLIER_ADDRESS"
+										},
+										"TAX": {
+											"type": "string",
+											"description": "TAX"
+										},
+										"TOTAL": {
+											"type": "string",
+											"description": "TOTAL"
+										},
+										"VENDOR_NAME": {
+											"type": "string",
+											"description": "VENDOR_NAME"
+										},
+										"VENDOR_VAT_NUMBER": {
+											"type": "string",
+											"description": "VENDOR_VAT_NUMBER"
+										},
+										"name": {
+											"type": "string",
+											"description": "name"
+										}
+									},
+									"description": "outputs"
+								}
+							}
+						}
+					}
+				},
+				"summary": "Extract data from Invoice",
+				"description": "Extract data from provisioned Invoice using predefined Bitskout plugin",
+				"operationId": "Extract_Data_From_Invoice",
```

Comment:
@saferoombiz all operationIds should be in Camel Case. please change to "ExtractDataFromInvoice".
Please fix all operationIds in the swagger file.